SETUP.md

```diff
@@ -0,0 +1,56 @@
+# Viam + Python
+
+This guide will help you get set up with Viam with Python. It assumes that you are starting from scratch, and will walk you through setting up a fresh environment and installing the necessary requirements.
+
+## Setup your project
+
+The first step is to create a directory to house your project. For this guide, we will be using the directory name `viam-python`:
+
+```bash
+mkdir viam-python
+cd viam-python
+```
+
+## Create a Virtual Environment
+
+Now that we are in the project directory, let's create and activate a virtual environment for python to run in.
+
+> **INFO**
+> Creating a virtual environment (`venv`) is important as it isolates this python environment from any other you might already have. This allows us to ensure a clean project and easier dependency management, as well so not bloating your global python environment.
+
+```bask
+python3 -m venv viam-env
```

Comment:
Does venv come by default with pip3?

Reply:
I tested it on various docker instances and it turns out that ubuntu doesn't come with the necessary requirements (at least not when I run it in docker). So I added a note below to guide users